docs/analyzers-creation.md

```diff
@@ -0,0 +1,74 @@
+# Implementing Your Own Analyzer
```

Comment:
I wish we had links to examples somewhere in this document. Because after reading it I still have no idea how to start.

Reply:
Was it improved with the provided examples?
eb222fb

Reply:
yes, definitely! Though python example is incorrect (it exits after 60 seconds).

Reply:
I think it's up to the user to decide how to proceed to wait before stopping the server.

I also tried to be concise, so I preferred not to handle that in detail.
On the other hand, it is also explained in the gRPC.io docs about how to start the server, already linked in our docs just above the python example.